src/components/editor/InputListLOC.jsx

```diff
@@ -50,15 +50,13 @@ class InputListLOC extends Component {
   }
 
   render() {
-    let lookupUri, isMandatory, isRepeatable
-    try {
-      isMandatory = JSON.parse(this.props.propertyTemplate.mandatory)
-      isRepeatable = JSON.parse(this.props.propertyTemplate.valueConstraint.repeatable)
-      lookupUri = this.props.lookupConfig.uri
-    } catch (error) {
-      console.error(`Some properties were not defined in the json file: ${error}`)
+    if (this.props.lookupConfig.uri === undefined) {
+      alert(`There is no configured list lookup for ${this.props.propertyTemplate.propertyURI}`)
     }
 
+    const isMandatory = booleanPropertyFromTemplate(this.props.propertyTemplate, 'mandatory', false)
+    const isRepeatable = booleanPropertyFromTemplate(this.props.propertyTemplate.valueConstraint, 'repeatable', true)
```

Comment:
@mjgiarlo the test was wrong as it was referencing the wrong repeatable value.

Reply:
ah, OK